beacon-chain/interop-cold-start/service.go

```diff
@@ -123,8 +133,8 @@ func (s *Service) ChainStartEth1Data() *ethpb.Eth1Data {
 }
 
 // PreGenesisState returns an empty beacon state.
-func (s *Service) PreGenesisState() *pb.BeaconState {
-	return &pb.BeaconState{}
+func (s *Service) PreGenesisState() *stateTrie.BeaconState {
+	return &stateTrie.BeaconState{}
```

Comment:
This should always be initialized from a protobuf for now, won't this cause issues?